scene/gui/split_container.cpp

```diff
@@ -299,24 +294,34 @@ void SplitContainer::_notification(int p_what) {
 		case NOTIFICATION_LAYOUT_DIRECTION_CHANGED: {
 			queue_sort();
 		} break;
-
 		case NOTIFICATION_SORT_CHILDREN: {
 			_resort();
 		} break;
-
 		case NOTIFICATION_THEME_CHANGED: {
 			update_minimum_size();
 		} break;
+		case NOTIFICATION_DRAW: {
+			draw_style_box(theme_cache.split_bar_background, split_bar_rect);
+			if (show_drag_area && Engine::get_singleton()->is_editor_hint()) {
+				draw_rect(dragging_area_control->get_rect(), dragging_enabled ? Color(1, 1, 0, 0.3) : Color(1, 0, 0, 0.3));
```

Comment:
This doesn't show above the second child of the split container when it has a minimum grab thickness. It should probably be moved to the dragger's draw notification or somewhere else.

Reply:
Great catch! Done.